C/c4CAPI.cc

```diff
@@ -484,10 +484,13 @@ C4Database* c4db_openNamed(C4String name,
 }
 
 
-C4Database* c4db_openAgain(C4Database* db,
+C4Database* c4db_openAgain(C4Database* database,
                            C4Error *outError) noexcept
 {
-    return c4db_openNamed(c4db_getName(db), c4db_getConfig2(db), outError);
+    if ( database == nullptr ) return nullptr;
```

Comment:
`database` is not nullable.  I checked the other places in the file and when the database is not nullable, I didn't see why have try to specifically handle that case.

Reply:
I guess I don't have to, but not-nullable is not runtime checked in the release, I think.